src/utils.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports.shuffle = (someArray) => {
```

Comment:
```suggestion
const shuffle = (items) => {
```
экспорты должны быть только внизу файла (исправить везде)

в переменных не должно быть типа данных